sci-geosciences/grass/grass-9999.ebuild

```diff
@@ -3,14 +3,14 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{6..9} )
+PYTHON_COMPAT=( python3_{8,9} )
 PYTHON_REQ_USE="sqlite"  # bug 572440
 WANT_AUTOCONF="2.1"
```

Comment:
This fails with 
```
***** autoconf *****
***** PWD: /var/tmp/portage/sci-geosciences/grass-9999/work/grass-9999
***** autoconf

ac-wrapper: Since configure.ac is present, aclocal always use
    autoconf 2.59+, which conflicts with your choice and
    causes error. You have two options:
    1. Try execute command again after removing configure.ac
    2. Don't set WANT_AUTOCONF
```

Reply:
Thanks I overlooked it. GRASS GIS upgrade [autoconf version to 2.69 ](https://github.com/OSGeo/grass/commit/dca9364c64d7dac5b4ae9262f016708a97c979d7) in last commits. I've already fixed it.